tests/lib/rules/no-constant-condition.js

```diff
@@ -166,6 +166,19 @@ ruleTester.run("no-constant-condition", rule, {
         { code: "for(;true;);", options: [{ checkLoops: false }] },
         { code: "do{}while(true)", options: [{ checkLoops: false }] },
 
+        // { checkLoops: "none" }
+        { code: "while(true);", options: [{ checkLoops: "none" }] },
+        { code: "for(;true;);", options: [{ checkLoops: "none" }] },
+        { code: "do{}while(true)", options: [{ checkLoops: "none" }] },
+
+        // { checkloops: "allExceptWhileTrue" }
+        { code: "while(true);", options: [{ checkLoops: "allExceptWhileTrue" }] },
```

Comment:
as it's the new default, good to have a test like:
```js
{ code: "while(true);"}
```

Reply:
Added!